src/status_im2/contexts/chat/messages/avatar/view.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im2.contexts.chat.messages.avatar.view
+  (:require [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [quo2.core :as quo]
+            [status-im2.contexts.chat.messages.avatar.style :as style]))
+
+(defn avatar [from size]
```

Comment:
"Connected" component https://react-redux.js.org/api/connect , so it can be reused in multiple places without having to subscribe everywhere